unity-renderer/Assets/DCLServices/PlacesAPIService/PlacesAPIService.cs

```diff
@@ -0,0 +1,232 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.Tasks;
+using DCLServices.Lambdas;
+using MainScripts.DCL.Controllers.HotScenes;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+
+namespace DCLServices.PlacesAPIService
+{
+    public interface IPlacesAPIService: IService
+    {
+        UniTask<(IReadOnlyList<IHotScenesController.PlaceInfo> places, int total)> GetMostActivePlaces(int pageNumber, int pageSize, CancellationToken ct, bool renewCache = false);
+
+        UniTask<IHotScenesController.PlaceInfo> GetPlace(Vector2Int coords, CancellationToken ct, bool renewCache = false);
+
+        UniTask<IHotScenesController.PlaceInfo> GetPlace(string placeUUID, CancellationToken ct, bool renewCache = false);
+
+        UniTask<IReadOnlyList<IHotScenesController.PlaceInfo>> GetFavorites(CancellationToken ct, bool renewCache = false);
+
+        UniTask SetPlaceFavorite(string placeUUID, bool isFavorite, CancellationToken ct);
+        UniTask SetPlaceFavorite(Vector2Int coords, bool isFavorite, CancellationToken ct);
+
+        UniTask<bool> IsFavoritePlace(IHotScenesController.PlaceInfo placeInfo, CancellationToken ct, bool renewCache = false);
+        UniTask<bool> IsFavoritePlace(Vector2Int coords, CancellationToken ct, bool renewCache = false);
+        UniTask<bool> IsFavoritePlace(string placeUUID, CancellationToken ct, bool renewCache = false);
+    }
+
+    public class PlacesAPIService : IPlacesAPIService, ILambdaServiceConsumer<IHotScenesController.PlacesAPIResponse>
+    {
+        private readonly IPlacesAPIClient client;
+
+        internal readonly Dictionary<int, LambdaResponsePagePointer<IHotScenesController.PlacesAPIResponse>> activePlacesPagePointers = new ();
+        internal readonly Dictionary<string, IHotScenesController.PlaceInfo> placesById = new ();
+        internal readonly Dictionary<Vector2Int, IHotScenesController.PlaceInfo> placesByCoords = new ();
+
+        //Favorites
+        internal bool composedFavoritesDirty = true;
```

Comment:
I don't see it is covered by tests, though exposed as `internal`. Didn't you consider adding them or maybe just switching to `private` otherwise?

Reply:
You are right, I considered adding it and forgot. Will add it